packages/config/config/devices/0x031e/lzw30-sn.json

```
@@ -352,6 +353,7 @@
 		"51": {
 			"$if": "firmwareVersion >= 1.19",
 			"label": "Instant On",
+			"description": "Controls the 700ms button delay. Note, if you disable the delay, it will also disable multi-tap scenes.",
```

```suggestion
			"description": "Enabling this disables the 700ms button delay and multi-tap scenes.",
```
Seems a bit less confusing to me.